### PR TITLE
First major implementation

### PR DESCRIPTION
**Pull request by Jata2k**
Resolves: DVOP-1603

### CODE REVIEW

File: groups/bootstrap/locals.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+data "vault_generic_secret" "secrets" {
+  path = "team-${var.team}/${var.account_name}/${var.region}/${var.repository_name}"
```
Comment (DrewHarvard) [inline groups/bootstrap/locals.tf:2]: You'll have to add `${var.environment}` to this path

File: groups/logging/locals.tf [hunk 2]
```diff
@@ -0,0 +1,21 @@
+data "vault_generic_secret" "secrets" {
+  path = "team-${var.team}/${var.account_name}/${var.region}/${var.repository_name}"
```
Comment (DrewHarvard) [inline groups/logging/locals.tf:2]: You'll have to add `${var.environment}` to this path

File: groups/shared/locals.tf [hunk 3]
```diff
@@ -0,0 +1,18 @@
+data "vault_generic_secret" "secrets" {
+  path = "team-${var.team}/${var.account_name}/${var.region}/${var.repository_name}"
```
Comment (DrewHarvard) [inline groups/shared/locals.tf:2]: You'll have to add `${var.environment}` to this path

**Review by marcransome — CHANGES_REQUESTED** [review]

File: groups/bootstrap/cloud-init.tf [hunk 4]
```diff
@@ -0,0 +1,45 @@
+data "template_cloudinit_config" "config" {
+  gzip          = true
+  base64_encode = true
+
+  part {
+    content_type = "text/cloud-config"
+    content = templatefile("${path.module}/cloud-init/templates/system-config.yml.tpl", {
+      instance_hostname = "${var.service}-${var.environment}-bootstrap.${local.dns_zone_name}"
```
[the remark this answers is not in the document]
Comment (Jata2k) [reply groups/bootstrap/cloud-init.tf:8]: Good point. I'll introduce this into a subsequent PR I think as I'll be rebuilding the stack once more in development anyway. That's probably as good a time as any to add this change

**Review by marcransome — APPROVED** [review]

**Review by marcransome — DISMISSED** [review]

**Pull request merged**